requirements.txt

```diff
@@ -1,37 +1,37 @@
-authutils==3.1.1
+authutils==4.0.0
 boto==2.36.0
 cryptography==2.3
-dictionaryutils==2.0.7
+dictionaryutils==3.0.0
 envelopes==0.4
 Flask==1.1.1
 flask-cors==3.0.3
 Flask-SQLAlchemy-Session==1.1
 fuzzywuzzy==0.6.1
 gen3authz==0.2.1
 graphene==2.0.1
-rx==1.6.1
 jsonschema==2.5.1
-lxml==3.8.0
+lxml==4.4.2
 pbr==2.0.0
 psycopg2==2.7.3.2
 python-keystoneclient==1.8.1
 PyYAML==5.1
 requests==2.22.0
 setuptools==30.1.0
+six==1.12.0
 simplejson==3.8.1
 sqlalchemy==1.3.5
 Werkzeug==0.16.0
 xmltodict==0.9.2
 more-itertools==5.0.0
--e git+https://git@github.com/uc-cdis/cdis_oauth2client.git@0.1.3#egg=cdis_oauth2client
-cdispyutils==0.2.13
-datamodelutils==0.4.7
-psqlgraph==2.0.2
+cdis_oauth2client==1.0.0
+cdispyutils==1.0.3
+datamodelutils==1.0.0
 # required for gen3datamodel, not required for sheepdog
--e git+https://git@github.com/uc-cdis/cdiserrors.git@0.1.1#egg=cdiserrors
--e git+https://git@github.com/uc-cdis/cdislogging.git@master#egg=cdislogging
--e git+https://git@github.com/NCI-GDC/cdisutils.git@f54e393c89939b2200dfae45c6235cbe2bae1206#egg=cdisutils
-gen3dictionary==2.0.0
-gen3datamodel==2.0.2
--e git+https://git@github.com/uc-cdis/indexclient.git@1.6.2#egg=indexclient
--e git+https://git@github.com/uc-cdis/storage-client.git@0.1.7#egg=storageclient
+psqlgraph==3.0.0
+cdiserrors==0.1.2
+cdislogging==1.0.0
+git+https://git@github.com/NCI-GDC/cdisutils.git@f54e393c89939b2200dfae45c6235cbe2bae1206#egg=cdisutils
+gen3dictionary==2.0.1
+gen3datamodel==3.0.1
+git+https://git@github.com/uc-cdis/indexclient.git@2.0.0#egg=indexclient
+git+https://git@github.com/uc-cdis/storage-client.git@0.1.7#egg=storageclient
```

Comment:
update to latest indexclient? https://github.com/uc-cdis/indexclient/releases/tag/2.0.0

we probably need py3 storageclient? https://github.com/uc-cdis/storage-client/releases/tag/1.0.0
or just remove the dependency if not used